### ISSUE TITLE
Pass in __dirname, and receive index.json but with paths extended

### ISSUE THREAD

**Issue by ArtskydJ** [ArtskydJ/test-audio#4]
## v2.0.0

Instead of the wonky `path.resolve` stuff, do `var files = require('test-audio')(__dirname)`

`files` should have paths and metadata:

``` js
[{
    name: 'test1.ogg',
    bytes: 32652,
    type: 'ogg',
    path: 'C:\Github\package\node_modules\test-audio\test_1.ogg',
    metadata: {
        artist: 'Joseph Dykstra',
        album: 'Greatest Hits'
    }
}, {
    ...
}]
```

**Comment by ArtskydJ** [comment]
Passing is `__dirname` is not needed. `var files = require('test-audio')()` will be sufficient.

**Comment by ArtskydJ** [comment]
Fixed in c501e7a49301e4485f6f2c08af7118e35e9943f.
